src/Symfony/Component/Notifier/Transport/TransportInterface.php

```diff
@@ -24,7 +25,7 @@ interface TransportInterface
     /**
      * @throws TransportExceptionInterface
      */
-    public function send(MessageInterface $message): void;
+    public function send(MessageInterface $message): SentMessage;
```

Comment:
BC-breaks should be documented

Reply:
Yes of course. Thank you @Koc! I wanted some feedback about the *how* before.